src/components/Stake.svelte

```diff
@@ -1,19 +1,24 @@
 <script lang="ts">
-  import { dotApi, storeCurrentAction, storeToken } from '$lib/stores';
+  import { dotApi, storeToken } from '$lib/stores';
   import type { ApiPromise } from '@polkadot/api';
   import { DOLLARS, submitStake, type TxnFinishedCallback } from '$lib/connections';
-  import { ActionForms, defaultDotApi } from '$lib/storeTypes';
-  import KeySelection from './KeySelection.svelte';
+  import { defaultDotApi } from '$lib/storeTypes';
   import { onMount } from 'svelte';
   import { isFunction } from '@polkadot/util';
-  import { isLocalhost } from '$lib/utils';
+  import { formatAccount, isLocalhost } from '$lib/utils';
   import TransactionStatus from './TransactionStatus.svelte';
   import type { web3Enable, web3FromSource } from '@polkadot/extension-dapp';
-  import type { AccountMap } from '$lib/polkadotApi';
+  import DropDownMenu from './DropDownMenu.svelte';
+  import Modal from './Modal.svelte';
+  import { user } from '$lib/stores/userStore';
+  import { Account, providerAccountsStore } from '$lib/stores/accountsStore';
+  import BlockSection from './BlockSection.svelte';
+
+  export let isOpen = true;
```

Comment:
Default to false so that there is no chance of it being open when not expected.

Reply:
fixed